docs/plugins.adoc

```diff
@@ -13,6 +13,18 @@ include::../plugins.yaml[tag="external_plugins"]
 organization. Be aware however, that adding an external plugin both to the repository and its organization link:https://github.com/kubernetes/test-infra/blob/7de525b1f6943e5d08d9a127b0b668cec404c665/prow/plugins/plugins_test.go#L143[will result in an error].
 <2> You can limit events dispatched by hook to your plugin.
 
+==== Creating custom plugin hint messages
+
+To enable customized plugin hint comment messages edit plugin's configuration file (eg. test-keeper.yaml) adding the `plugin_hint` property.
+
+[source, yml, indent=0]
+----
+include::../../pkg/plugin/test-keeper/plugin/test_fixtures/github_calls/prs/with_tests/test-keeper.yml[tag="plugin_hint"]
+----
+
+The `plugin_hint` property can either have an inline string directly written inside of the config file or another file with case insensitive 
+filename PLUGIN-NAME-HINT.md that contains the hint. The location of the file can be set by either a relative path or an
```

Comment:
in the documentation, you say that the expected filename is `PLUGIN-NAME-HINT.md`, but in the code
https://github.com/arquillian/ike-prow-plugins/pull/84/files#diff-104213454f0a74d63ea38f4cc491246bR36
you match anything that is ending with `.md` suffix. Please, make it unified.

Reply:
I am slightly confused with the naming convention here. We say that `plugin-hint` is a common property for plugins and not specific or limited to just the test-keeper plugin. I understand that at the moment that is the only plugin we use this for. 

But thinking about it, if we have more than 1 plugin using it, can we have a common `plugin-hint` file being shared across the plugins and if so, should we then tie this file with the `plugin-name`?

Also, if that is not the case then I have two concerns:
1. Should we then move the plugin_name to the configuration because the comment file where we have the regex defined has no way of identifying the plugin-name? 
2. We need to update the plugin_hint files used in the project since they to do abide by this convention.   

What are your thoughts on this @MatousJobanek @bartoszmajsak?

Reply:
Each plugin has a name, so having `abcd` and `wip` as names the expectation was to have:

- `abcd_hint.md`
- `wip_hint.md`

I think it would be good to just add a real example in the doc for `test-keeper`.

> But thinking about it, if we have more than 1 plugin using it, can we have a common plugin-hint file being shared across the plugins and if so, should we then tie this file with the plugin-name?

I believe intention here is to have plugin specific hints. How to use given plugin. Not sure we want to have something to be used across all the plugins the same way - if you have any concrete example don't hesitate to point that out.

> We need to update the plugin_hint files used in the project since they to do abide by this convention. 

we only have them in the tests right now, don't we?